Jenkinsfile

```diff
@@ -78,6 +78,22 @@ def runCleanupResource(String backend){
     """
 }
 
+def runRestoringMonitoringStack(){
+    sh """
+    #!/bin/bash
+
+    set -xe
+    env
+
+    export MONITOR_STACK_TEST_ID=`cat latest/test_id`
+    echo "Restoring Monitor stack for test-id \$MONITOR_STACK_TEST_ID"
+    ./docker/env/hydra.sh investigate show-monitor \$MONITOR_STACK_TEST_ID
+    restoring_status = \$?
+    docker rm -f aprom agraf aalert
```

Comment:
can you move the cleanup to the pipeline ?  or at least to the try/finally clause, since it should be clean regardless ?

Reply:
also there would be an issue, when using this at the same time on the same worked, the port would be taken and it would fail

Reply:
another issue here is same docker name.

Reply:
Fixed, dockers now will have name-port(monitorstack feature), this will prevent failing by same name,
also dockers running on different port,this allow to run monitor stack for different commits on same host
also added the new parameter to command investigate show-monitor, which allow to kill appropriate dockers. mostly needed for test purpose in CI.